changelogs/unreleased/fix-journal-balance-html-tag-no-longer-computed.yml

```diff
@@ -0,0 +1,3 @@
+---
+title: "JOURNAL : Balance HTML tag no longer computed"
```

Comment:
Not explicit enough (can be interpretated as "Balance is no longer computed")
Title can be: "JOURNAL: Balance HTML tag is now correcly computed"